sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -1930,6 +1930,23 @@ pi_result piextUSMEnqueueMemAdvise(pi_queue, const void *, size_t,
   DIE_NO_IMPLEMENTATION;
 }
 
+pi_result piextUSMEnqueueFill2D(pi_queue, void *, size_t, size_t, const void *,
```

Comment:
What about get info for ESIMD Emulator plugin?

Reply:
`piContextGetInfo` isn't implemented in the esimd_emulator, so I assume it takes another route.